botocore/signers.py

```diff
@@ -196,6 +208,65 @@ def generate_presigned_url(self, request_dict, expires_in=3600,
         return request.url
 
 
+class CloudFrontSigner(object):
+
+    @staticmethod
```

Comment:
I would recommend to NOT remove it. When this [special base64 may be as simple as a one-liner in other language such as PHP](https://github.com/aws/aws-sdk-php/blob/master/src/CloudFront/UrlSigner.php#L95), in Python it needs special care to handle different strings in Python 2 and Python 3. It bit me, and I ended up developing dedicated test case for it. So I think it deserve to become a separated method.

I will adopt your suggestion to demote it to an internal function _b64encode(). Thanks!